web/thesauruses/python/functions.json

```diff
@@ -0,0 +1,85 @@
+{
+  "meta": {
+    "language": "python",
+    "language_version": "3.7",
+    "language_name": "Python"
+  },
+  "categories": {
+    "Void Functions": [
+      "void_function_no_parameters",
+      "void_function_with_parameters",
+      "void_function_variable_parameters"
+    ],
+    "Return Value Functions": [
+      "return_value_function_no_parameters",
+      "return_value_function_with_parameters",
+      "return_value_function_variable_parameters"
+    ],
+    "Lambda/Anonymous Functions": [
+      "anonymous_function_no_parameters",
+      "anonymous_function_with_parameters",
+      "anonymous_function_variable_parameters"
+    ],
+    "Subroutines": [
+      "call_subroutine",
+      "return_from_subroutine"
+    ]
+  },
+  "functions": {
+    "void_function_no_parameters": {
+      "name": "Function that does not return a value and takes no parameters",
+      "code": "
+      def function_name():
+        statements"
+    },
+    "void_function_with_parameters": {
+      "name": "Function that does not return a value and that takes 1 or more defined parameters",
+      "code": "
+      def function_name(parameter, parameter_1 ...):
```

Comment:
Can you remove the `...`? It might imply you actually include it or this can take an undefined number of parameters (which some language actually use the `...` for this)